projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/JobsService.java

```diff
@@ -240,9 +240,8 @@ public void updateLastExecution(
          return;
       }
 
-      dataJob.setLastExecutionStatus(dataJobExecution.getStatus());
-      dataJob.setLastExecutionEndTime(dataJobExecution.getEndTime());
```

Comment:
Why removing these?

Reply:
I figured they were not needed. The query will update the respective fields directly given the job name.

Reply:
Gotcha.